content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,20 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Support for the `attributes` field in Alerta node.
+- Add BigPanda handler options.
```

Comment:
Do we need to update the following docs to reflect these new configuration options/field available in event handlers?:

- [BigPanda event handler configuration](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/bigpanda/) 
- [Alerta event handler](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/alerta/)

Reply:
@kelseiv I was thinking the same thing when working on this

Reply:
Maybe @docmerlin can help us clarify

Reply:
I believe the answer is yes.

Reply:
Update BigPanda text to better reflect the change -- >Add "host" and "attribute" options to BigPanda node and "auto-attributes" configuration option.

We should update the examples for using the Alerta handler and the BigPanda handler to demonstrate how to leverage these two new improvements.

Reply:
review details for the BigPanda improvements here: https://github.com/influxdata/kapacitor/pull/2643

Reply:
create issue https://github.com/influxdata/docs-v2/issues/3707 for BigPanda and Alerta event handler improvement/example updates